src/styles/main.scss

```diff
@@ -1,3 +1,72 @@
+$cell-size: 100px;
+
+html {
+  font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;
+}
+
 body {
   margin: 0;
 }
+
+.field {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: $cell-size * 7 + 6px;
```

Comment:
Change the width of the .calendar to include 10px padding on both sides, i.e. $cell-size * 7 + 16px.